languageServer/src/LanguageServer.ts

```diff
@@ -10,16 +10,14 @@ import {
   Location,
```

Comment:
File definitions inside of `cmd`s don't seem to work correctly on my end: 

https://user-images.githubusercontent.com/43496356/194176392-50d8b562-08e6-46bd-b3bd-ddada358f3b1.mov